app/controllers/account_users_controller.rb

```diff
@@ -47,6 +48,7 @@ def destroy
     @account_user.deleted_by = session_user.id
 
     if @account_user.save
```

Comment:
I think I'd probably call the method `soft_delete`, but I don't really have a strong feeling about it.